docs: adjust readme according to suggestions (#577)
jina-ai/docarray: README.md

left_da = DocumentArray.pull('demo-leftda', show_progress=True)
```

**Note**

> If you have at least XGB of RAM and want to try using the whole dataset instead of just the first 1000 images, remove [:1000] when loading the files into the DocumentArrays left_da and right_da.

Reviewer: Should "XGB" be a specific number of gigabytes of RAM? "At least XGB" doesn't make sense to me.

Author: yes, I am trying to figure X out

> You will need more than 15GB of RAM if you want to try using the whole dataset instead of just the first 1000 images, remove [:1000] when loading the files into the DocumentArrays left_da and right_da.

Reviewer: ```suggestion
If you have more than 15GB of RAM and want to try using the whole dataset instead of just the first 1000 images, remove [:1000] when loading the files into the DocumentArrays left_da and right_da.
```

You will see a running progress bar to indicate the downloading process.

> To get a feeling of the data you will handle, plot them in one sprite image, you will need to have matplotlib and torch installed to run this snippet:

Reviewer: ```suggestion
To get a feeling of the data you will handle, plot them in one sprite image. You will need to have matplotlib and torch installed to run this snippet:
```
